user-guide/Advanced_Modules/Dashboards_legacy/Legacy_dashboards.md

```diff
@@ -17,9 +17,7 @@ DMS Dashboards is a legacy module that allows you to design and view customized
 - [Legacy Dashboard Gateway installation](xref:Legacy_Dashboard_Gateway_installation)
 
 > [!NOTE]
->
-> - For more information on the Dashboards web app available from DataMiner 9.6.9 onwards, see [Dashboards app](xref:newR_D#dashboards-app).
-> - From DataMiner 10.2.0/10.1.12 onwards, the legacy Dashboards module can be disabled using the soft-launch option *LegacyReportsAndDashboards*. See [Soft-launch options](xref:SoftLaunchOptions).
+> This module is by default disabled from DataMiner 10.4.0/10.4.1 onwards.<!-- RN 37786 --> If you want to keep using the legacy Dashboards module, you need to enable it with the [*LegacyReportsAndDashboards* soft-launch option](xref:Overview_of_Soft_Launch_Options#legacyreportsanddashboards). Prior to this, starting from DataMiner 10.2.0/10.1.12, the legacy Dashboards module can optionally be disabled with this soft-launch option.
```

Comment:
disabled by default instead of by default disabled (same applies to the other pages)?

Reply:
Both are possible; "by default disabled" puts the emphasis on the fact that it happens by default, while "disabled by default" is more neutral. In this case, this emphasis makes sense because it could optionally be disabled previously.